sites/cnfpt/src/backend/cnfpt/settings.py

```diff
@@ -464,6 +471,32 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
     RICHIE_ES_INDICES_PREFIX = values.Value(
         default="richie", environ_name="RICHIE_ES_INDICES_PREFIX", environ_prefix=None
     )
+    RICHIE_ES_STATE_WEIGHTS = values.ListValue(None)
+
+    # LTI Content
+    RICHIE_LTI_PROVIDERS = {
+        "lti_provider_test": {
+            "oauth_consumer_key": values.Value(
+                "InsecureOauthConsumerKey",
+                environ_name="LTI_TEST_OAUTH_CONSUMER_KEY",
+                environ_prefix=None,
+            ),
+            "shared_secret": values.Value(
+                "InsecureSharedSecret",
+                environ_name="LTI_TEST_SHARED_SECRET",
+                environ_prefix=None,
+            ),
+            "base_url": values.Value(
+                "https://lti.tools/saltire/tp",
```

Comment:
since you put the marsha name, put the marsha url pattern:
`https://marsha\.education/lti/videos/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}`